src/Tooltip.js

```diff
@@ -5,26 +5,28 @@ import PropTypes from 'prop-types';
 import 'react-tippy/dist/tippy.css';
 import { Tooltip as TippyTooltip } from 'react-tippy';
 
-import Paragraph from './Paragraph';
+import Text from './Text';
 
 // eslint-disable-next-line no-unused-expressions
 injectGlobal`
   .tippy-tooltip.arbor-theme {
     padding: 0;
+    border-radius: 3px;
+    max-width: 240px;
```

Comment:
Is 240px a number we'll be reusing for anything else? Also, @dgalarza should we switch them to rem?

Reply:
The 3px radius is one of our theme variables, but it wasn't working with injectGlobal (which we're using to override tippy styles). It'd be ideal to use our theme if we can.

Looks like react-tippy includes its own max-width so I'm fine keeping that.

Reply:
@mayravherman is right when it comes to the border-radius here. You can't use the `ThemeProvider` theme inside injectGlobal since it's out of scope of a component. However, we could at least import the theme right into this file and refer to the theme directly.

All this means is that if someone ever provided their own theme and changed that border radius theme property, their theme wouldn't be used but the default Arbor theme would be used instead. Not the end of the world IMO since either way, someone would need to use `injectGlobal` to override these.